model/SecurityConfig.ts

```diff
@@ -1,8 +1,12 @@
+import { isly } from "isly"
+
 export interface SecurityConfig {
 	enforce2fa?: boolean
+	passwordExpirationDays?: number
 }
 export namespace SecurityConfig {
-	export function is(value: SecurityConfig | any): value is SecurityConfig {
-		return typeof value == "object" && (typeof value.enforce2fa == "boolean" || value.enforce2fa == undefined)
-	}
+	export const type = isly.object<SecurityConfig>({
+		enforce2fa: isly.boolean().optional(),
+		passwordExpirationDays: isly.number().optional(),
+	})
```

Comment:
export const is = type.is

Reply:
Unless you think we would stop doing that since we could just do `SecurityConfig.type.is`

Reply:
:+1: